src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftOAuth2SecurityRealm.java

```diff
@@ -448,6 +454,12 @@ boolean populateDefaults() throws IOException, GeneralSecurityException {
                     this.serverPrefix));
             LOGGER.info(format(msg5, this.defaultedServerPrefix));
         }
+
+        configMapName = EnvVars.masterEnvVars.get("CONFIG_MAP_NAME");
+        if (configMapName == null) {
+            configMapName = CONFIG_MAP_DEFAULT_NAME;
+        }
```

Comment:
I haven't tested the below code, but I think it conveys that I think we should set the configmapName to the default first, and then if the env var is set, change it to that value.
```suggestion
        configMapName = CONFIG_MAP_NAME_DEFAULT;
        if (EnvVars.masterEnvVars.get("CONFIG_MAP_NAME") != null) {
            configMapName = EnvVars.masterEnvVars.get("CONFIG_MAP_NAME");
        }
```